buildSrc/src/main/groovy/org/elasticsearch/gradle/LoggedExec.java

```diff
@@ -0,0 +1,44 @@
+package org.elasticsearch.gradle;
+
+import groovy.lang.Closure;
+import org.codehaus.groovy.runtime.DefaultGroovyMethods;
+import org.codehaus.groovy.runtime.StringGroovyMethods;
+import org.gradle.api.GradleException;
+import org.gradle.api.Task;
+import org.gradle.api.tasks.Exec;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.stream.Collectors;
+
+/**
+ * A wrapper around gradle's Exec task to capture output and log on error.
+ */
+public class LoggedExec extends Exec {
```

Comment:
This appears to be moved to a java file, but still exists under `src/main/groovy`?